script/Arcueid/Day 6/06_00_ARC06_3_1.txt

```diff
@@ -49,7 +49,7 @@ Though, perhaps it would be more accurate to say that it was also her boundary.
 [sha:3a3ad51a874d8e831887a34807a5b8baf1fd7eef]{
 -- Page 263, Offset 14547.
 -- 秋葉ひとりでは園まで出る事はなく、このテラスまでがあいつの移動範囲だった。
-She never went out into the garden by herself, this terrace marked the confines of where she was allowed to roam.
+Since she never went out into the garden by herself, this terrace marked the confines of where she was allowed to roam.
```

Comment:
This isn't really causative here in JP, more two statements of fact - I guess you could change the comma to a semicolon if you wanted to make a change here?